config/initializers/prepends.rb

```diff
@@ -0,0 +1 @@
+AttachFilesToWorkJob.prepend(AttachFilesToWorkJobPrepends)
```

Comment:
Could we use this same strategy to add additional logging to Hyrax background jobs?

Reply:
Yeah, and doing that way avoids needing to override it which could be problematic if someone is already overriding a job.